src/libs/ReportUtils.js

```diff
@@ -2109,14 +2109,14 @@ function shouldReportBeInOptionList(report, currentReportId, isInGSDMode, iouRep
 
 /**
  * Attempts to find a report in onyx with the provided list of participants. Does not include threads
- * @param {Array} newParticipantList
+ * @param {Array<Number>} newParticipantList
  * @returns {Array|undefined}
  */
 function getChatByParticipants(newParticipantList) {
     newParticipantList.sort();
     return _.find(allReports, (report) => {
         // If the report has been deleted, or there are no participants (like an empty #admins room) then skip it
-        if (!report || !report.participantAccountIDs || isChatThread(report)) {
```

Comment:
`!report.participantAccountIDs` doesn't work if `participantAccountIDs` is an empty array so i thinkkkkk we want to actually check length here to be safe